<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pharmacy_newstock</class>
 <widget class="QDialog" name="Pharmacy_newstock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Drugs</string>
  </property>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>532</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(173, 235, 173)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Medicine Description</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_md">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Search Drug</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>342</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>MedicineName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_name_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>801</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fill">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>260</y>
     <width>148</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txt_mid"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_msn"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_cost"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_company"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_qs"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_ql"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_nsd"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_contact"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>261</y>
     <width>129</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>MedicineID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Medicine Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Medicine Serial No</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Company</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Quantity Supplied</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Quantity Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Next Supply Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Contact</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
